www/js/submit.js

```diff
@@ -0,0 +1,17 @@
+
+$(document).ready(function(){
+  var $submitButton = $(".submit-button");
+  $submitButton.click(function() {
+     var ask = window.confirm("You are offline right now, ready to store it?");
+         if (ask) {
+                window.alert("This post was successfully stored.");
+                    
+                window.location = "homepage.html";
+                    
```

Comment:
correct me if I am wrong, but after you go to homepahe.html, wouldn't there still be two surveys pending? I think you should hide them with css somehow.